### PR TITLE
Fix up cross-compilation issues

### PR DESCRIPTION
**Pull request by vvuk**
Fix a few cross-compilation issues.  This won't actually be used with MSVC builds, but fixes Android issue.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/libfontconfig/22)

### CODE REVIEW

File: makefile.cargo [hunk 1]
```diff
@@ -23,6 +23,12 @@ AR ?= ar
 
 endif
 
+ifeq ("$(MSYSTEM)","")
+CXXFLAGS += -fPIC
```
Comment (metajack) [inline makefile.cargo]: Doesn't rust require -fPIC or -fPIE?

**Comment by metajack** [comment]
r=me once we resolve the -fPIC thing.

File: makefile.cargo (hunk 1 above)
Comment (vvuk) [inline makefile.cargo]: I'm pretty sure -fPIC is a no-op on Windows, due to the ABI.  This builds, at least.

**Comment by metajack** [comment]
@bors-servo r+

---

Reviewed 1 of 1 files at r1.
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/servo/libfontconfig/22#-:-KMRI6ITqQ9g6prFp3Fe:1236647940)*

**Pull request merged**